src/braket/experimental/autoqasm/instructions/gates.py

```diff
@@ -16,11 +16,15 @@
 
 from typing import Union
 
+import oqpy
+
 from braket.circuits.free_parameter_expression import FreeParameterExpression
 
 from .instructions import _qubit_instruction
 from .qubits import QubitIdentifierType
 
+GateParameterType = Union[float, FreeParameterExpression, oqpy._ClassicalVar]
```

Comment:
thanks for adding this!

Reply:
Is it possible to use AutoQASM `types` instead? We are trying to work towards minimizing `oqpy` imports

Reply:
We don't have an existing Union type here, but I can make one. I think it might be more fitting to address this together with other similar cases. I'm still in favor of adding the oqpy reference here to make sure this case gets lumped in with the others and not missed.